pkg/utils/stream_fields_parser.go

```diff
@@ -0,0 +1,67 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+)
+
+// StreamField represents the label values pair of the
+// _stream fields in the log struct
+type StreamField struct {
+	Label string
+	Value string
+}
+
+// ParseStreamFields collect labels values pair from the _stream field
+func ParseStreamFields(streamFields string) ([]StreamField, error) {
+	if streamFields == "" {
+		return nil, nil
+	}
+
+	if !strings.HasPrefix(streamFields, "{") {
+		return nil, fmt.Errorf("_stream field must start with '{' or '}'")
+	}
+	if !strings.HasSuffix(streamFields, "}") {
+		return nil, fmt.Errorf("_stream field must end with '}'")
+	}
+	if strings.HasSuffix(streamFields, `""}`) {
```

Comment:
weird check, the quote could be anywhere before `}`

Reply:
removed this check and test.